docs/MO_DG/prepare_model/Additional_Optimizations.md

```diff
@@ -2,9 +2,47 @@
 
 Input data for inference can be different from training dataset and requires additional preprocessing before inference.
 In order to accelerate the whole pipeline including preprocessing and inference, Model Optimizer provides special parameters such as `--mean_values`,
-`--scale_values` and `--reverse_input_channels`. Based on these parameters, Model Optimizer generates IR with additionally
+`--scale_values`, `--reverse_input_channels` and `--layout`. Based on these parameters, Model Optimizer generates IR with additionally
```

Comment:
comma is missed:  `--reverse_input_channels`, and `--layout

Reply:
comma before "and"? why?

Reply:
grammar rules it is a case with a list of more two items.

Reply:
Yep, it is not a strict rule and both versions should be considered "proper" but as a good practice it would be awesome if we used comma before "and" where there are more than two elements listed. It's called Oxford Comma by some, makes text easier to read, and is more popular in American English than in British English. Sorry for the long explanation but I'm a fan of Oxford Comma :) 
Examples:
I ate a hot-dog, a meatloaf and a tortilla. (hot-dog is not a meatloaf and a tortilla put together, as it might be understood).
We ate a hot-dog and Cola, pizza and Fanta, a burger and beer, and nachos and Sprite. (just imagine it without commas ;) )